<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>301</width>
      <height>401</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>Control Panel</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>310</y>
       <width>301</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>LIU Zhihao
(liuzh96@outlook.com)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>301</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="ckbBVHS">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Debug: BVH Structure</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ckbWireFrame">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>WireFrame / Shaded</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_6">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Inverse Y</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: 1px solid #8f8f91;
    border-radius: 10px;
    background-color: rgb(66,133,244);
    min-width: 80px;
    color:#FFFFFF;
}

QPushButton:hover{
    border: 1px solid #8f8f91;
    border-radius: 3px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #6265F4, stop: 1 #3265F4);
    min-width: 80px;
}

QPushButton :pressed{
    border: 1px solid #8f8f91;
    border-radius: 2px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #5295F4, stop: 1 #4285F4);
    min-width: 80px;
    color:#FFFFFF;
}


</string>
     </property>
     <property name="text">
      <string>Load Rugosity Data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: 1px solid #8f8f91;
    border-radius: 10px;
    background-color: rgb(194,67,53);
    min-width: 80px;
    color:#FFFFFF;
}

QPushButton:hover{
    border: 1px solid #8f8f91;
    border-radius: 3px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 rgb(180,110,110), stop: 1 rgb(110,10,10));
    min-width: 80px;
}

QPushButton :pressed{
    border: 1px solid #8f8f91;
    border-radius: 2px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #4E6060, stop: 1 #1B1B1B);
    min-width: 80px;
    color:#FFFFFF;
}


</string>
     </property>
     <property name="text">
      <string>Open PLY Mesh</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSaveRugosity">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: 1px solid #8f8f91;
    border-radius: 10px;
    background-color: rgb(66,133,244);
    min-width: 80px;
    color:#FFFFFF;
}

QPushButton:hover{
    border: 1px solid #8f8f91;
    border-radius: 3px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #6265F4, stop: 1 #3265F4);
    min-width: 80px;
}

QPushButton :pressed{
    border: 1px solid #8f8f91;
    border-radius: 2px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #5295F4, stop: 1 #4285F4);
    min-width: 80px;
    color:#FFFFFF;
}


</string>
     </property>
     <property name="text">
      <string>Save Rugosity Data</string>
     </property>
    </widget>
   </widget>
   <widget class="QtCharts::QChartView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>431</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QtCharts::QChartView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>431</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>787</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
